### ISSUE TITLE
Going really slow through a portal completely ignores the portal entity

### ISSUE THREAD

**Issue by TahoeBennie** [qouteall/ImmersivePortalsMod#1005]
Essentially, going really slow (like really slow) through a portal causes it to ignore the portal entirely.
To make it happen, set the mouse speed really slow so and hold W up against a wall that has a portal at it, and go as slow as possible into the portal, and eventually it will skip the portal when you get to the center point of it.

**Comment by qouteall** [comment]
Can you show a video about it?

**Comment by TahoeBennie** [comment]
Yes here's what I found with it, only way I can think of it being bad is when you are not meant to be able to access behind the portal and then get stuck there:
https://imgur.com/a/1JsJKXA

**Comment by qouteall** [comment]
Possibly related to some floating point inaccuracy
